rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :customer
```

Comment:
- [ ] the user can read and write `book`, and `customer (person)`, kindly make sure to add them to the atter_accessor instead